### ISSUE TITLE
Some question

### ISSUE THREAD

**Issue by panpandzf** [amvarani/Plant_Annotation_TEs#1]
In the full file in the result of mgescan, what do the suffixes en.dna and rt.dna stand for, and why are they used as candidate LINE sequences,some sequences have the same starting position. Why not extract the corresponding sequence according to the nontrl.gff3 file, please forgive my ignorance, looking forward to your reply.

**Comment by amvarani** [comment]
Indeed, you make a valid point. However, it is important to note that a clustering step occurs at the end of the primary library generation process, which effectively removes those smaller and contained sequences.
